extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -338,23 +341,36 @@ export class AmpStory360 extends AMP.BaseElement {
    */
   getPageId_() {
     if (this.pageId_ == null) {
-      this.pageId_ = closest(dev().assertElement(this.element), (el) => {
-        return el.tagName.toLowerCase() === 'amp-story-page';
-      }).getAttribute('id');
+      this.pageId_ = this.getPage_().getAttribute('id');
     }
     return this.pageId_;
   }
 
+  /**
+   * @private
+   * @return {!Element} the parent amp-story-page
+   */
+  getPage_() {
+    return closest(dev().assertElement(this.element), (el) => 
+      el.tagName.toLowerCase() === 'amp-story-page';
```

Comment:
Think I made this mistake in earlier suggestion.

```suggestion
      el.tagName.toLowerCase() === 'amp-story-page'
```